<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArenaUI</class>
 <widget class="QMainWindow" name="ArenaUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assisi|bf Arena UI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGraphicsView" name="arenaSpace">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
      </widget>
      <widget class="QSplitter" name="sideLayout">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabCasus">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <attribute name="title">
          <string>Casu data</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="opaqueResize">
             <bool>true</bool>
            </property>
            <widget class="QTreeWidget" name="casuTree">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::ActionsContextMenu</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>false</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="headerVisible">
              <bool>true</bool>
             </attribute>
             <attribute name="headerDefaultSectionSize">
              <number>200</number>
             </attribute>
             <attribute name="headerHighlightSections">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Value</string>
              </property>
              <property name="textAlignment">
               <set>AlignTrailing|AlignVCenter</set>
              </property>
             </column>
            </widget>
            <widget class="QTreeWidget" name="groupTree">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::ActionsContextMenu</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabTrend">
         <attribute name="title">
          <string>Trend graphs</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tabDeploy">
         <attribute name="title">
          <string>Deployment</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>159</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
               <string>Arena description file:</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="arenaLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>159</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
               <string>Neighborhood file:</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="neighLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>159</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text">
               <string>Deployment file:</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="deployLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="deployButtons" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="3" column="2">
              <widget class="QPushButton" name="_7stopSim">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Stop simulator</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="_1runButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Run</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QPushButton" name="_4collectButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Collect data</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="_5cleanButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Clean log</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="_6startSim">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Start simulator</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="_2stopButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="_3deployButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Deploy</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QDeploy" name="deployWidget" native="true"/>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>734</width>
               <height>466</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="clearOutput">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Clear output</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="toggleShellOut">
              <property name="text">
               <string>Catch shell output</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenArena"/>
   <addaction name="actionSave"/>
   <addaction name="actionGroup"/>
   <addaction name="actionUngroup"/>
   <addaction name="actionConnect"/>
   <addaction name="actionToggleLog"/>
   <addaction name="actionCamera"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionGroup">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/group.png</normaloff>:/images/icons/group.png</iconset>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
   <property name="toolTip">
    <string>Group selected CASUs (Ctrl+G)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionUngroup">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/ungroup.png</normaloff>:/images/icons/ungroup.png</iconset>
   </property>
   <property name="text">
    <string>Ungroup</string>
   </property>
   <property name="toolTip">
    <string>Ungroup selected CASUs (Ctrl+U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionOpenArena">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/open.png</normaloff>:/images/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open project or session (Ctrl + O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/connect.png</normaloff>:/images/icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to CASU (Ctrl+C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionToggleLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/log_off.png</normaloff>
     <normalon>:/images/icons/log_on.png</normalon>:/images/icons/log_off.png</iconset>
   </property>
   <property name="text">
    <string>toggleLog</string>
   </property>
   <property name="toolTip">
    <string>Toggle Log (Ctrl + L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionPlot_selected_in_same_trend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Plot selected in same trend</string>
   </property>
   <property name="toolTip">
    <string>(Ctrl + P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionPlot_selected_in_different_trends">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Plot selected in different trends</string>
   </property>
   <property name="toolTip">
    <string>(Ctrl + Shift + P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+P</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/settings.png</normaloff>:/images/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/save.png</normaloff>:/images/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save session (Ctrl + S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/camera.png</normaloff>:/images/icons/camera.png</iconset>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
   <property name="toolTip">
    <string>Toggle video stream (Ctrol + V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDeploy</class>
   <extends>QWidget</extends>
   <header>qdeploy.h</header>
   <container>1</container>
   <slots>
    <slot>deploy()</slot>
    <slot>run()</slot>
    <slot>stop()</slot>
    <slot>collect()</slot>
    <slot>cleanLog()</slot>
    <slot>toggleOutput(bool)</slot>
    <slot>simulatorStart()</slot>
    <slot>simulatorStop()</slot>
    <slot>clear()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>arenaSpace</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>_3deployButton</sender>
   <signal>pressed()</signal>
   <receiver>deployWidget</receiver>
   <slot>deploy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>937</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>869</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_7stopSim</sender>
   <signal>pressed()</signal>
   <receiver>deployWidget</receiver>
   <slot>simulatorStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>953</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>869</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_1runButton</sender>
   <signal>pressed()</signal>
   <receiver>deployWidget</receiver>
   <slot>run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>869</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_6startSim</sender>
   <signal>pressed()</signal>
   <receiver>deployWidget</receiver>
   <slot>simulatorStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>869</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_5cleanButton</sender>
   <signal>pressed()</signal>
   <receiver>deployWidget</receiver>
   <slot>cleanLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>869</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_4collectButton</sender>
   <signal>pressed()</signal>
   <receiver>deployWidget</receiver>
   <slot>collect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1125</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>869</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearOutput</sender>
   <signal>clicked()</signal>
   <receiver>deployWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>815</y>
    </hint>
    <hint type="destinationlabel">
     <x>869</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggleShellOut</sender>
   <signal>toggled(bool)</signal>
   <receiver>deployWidget</receiver>
   <slot>toggleOutput(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>922</x>
     <y>814</y>
    </hint>
    <hint type="destinationlabel">
     <x>869</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
